### PR TITLE
Implemented a max level for quests

### PR DESCRIPTION
**Pull request by GuybrushGit**
Fixes https://github.com/LightsHope/server/issues/568

### CODE REVIEW

**Comment by brotalnia** [comment]
Why are you disabling the past victories quests? They existed on retail, custom changes go to custom sql folder. The quests already have the correct patch values and are only available after 1.8 when Marks of Honor were added. They exist to let you exchange the old rewards (arathi resource crate, silverwing talisman, warsong gulch mark of honor) for the new mark of honor rewards. This is all fully implemented.

**Comment by GuybrushGit** [comment]
The old rewards are implemented? My bad I thought we had marks of honor through all the patches. I'll remove that line.

**Comment by brotalnia** [comment]
Yes i implemented them even before i merged the progression. On Lightbringer there is a custom patch in place to change rewards to Marks of Honor, because it was too late to switch to the old rewards out of nowhere in 1.7.

File: sql/migrations/20171114231424_world.sql [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+UPDATE `quest_template` SET `MaxLevel`=QuestLevel WHERE Type = 41;
+UPDATE `quest_template` SET `MaxLevel`=0 WHERE `entry` IN 
+(7842,8262,8260,8261,7838,7810,9665,1658,8265,8263,8264,9664,8848,8853,8846,8851,7841,7868,7866,7867,8322,8409,
+9419,9422,7865,7863,7864,1657,8847,8852,8509,8492,8494,8511,8517,8513,8510,8493,8495,8512,8518,8514,8506,8525,
+8527,8523,8521,8529,8504,8516,8500,8505,8524,8526,8522,8520,8528,8503,8515,8499,8795,7843,8615,8532,8580,8588,
+8611,8607,8545,8616,8533,8581,8589,8612,8608,8546,8550,8583,8601,8610,8614,8591,8543,8605,8549,8582,8600,8609,
+8613,8590,8542,8604,8792,8793,8794,8373,8850,8855,8849,8854,8001);
+
+-- Old versions of: Vanquish the Invaders!, Quell the Silverwing Usurpers, Talismans of Merit, Mark of Honor, Arathi Basin Resources!, More Resource Crates, Cut Arathor Supply Lines
+UPDATE `quest_template` SET `MaxLevel`=29 WHERE `ID`=7788;
```
Comment (thepparker) [inline sql/migrations/20171114231424_world.sql]: There is no `ID` field on quest_template, it's `entry`

Comment (brotalnia) [reply sql/migrations/20171114231424_world.sql]: oopsie

**Pull request merged**